include/aikido/control/BarrettFingerKinematicSimulationPositionCommandExecutor.hpp

```diff
@@ -82,10 +82,9 @@ class BarrettFingerKinematicSimulationPositionCommandExecutor
   /// is reached, or collision is detected.
   void step(const std::chrono::system_clock::time_point& timepoint) override;
 
-  /// Sets CollisionGroup to check against for finger collisions.
-  ///
-  /// \param collideWith CollisionGroup to check finger collisions
-  /// \return false if collideWith cannot be set (during execution)
+  /* clang-format off */
```

Comment:
Nit: Could we use `// clang-format off` (C++ style) instead of `/* clang-format off */` (C style)?